src/stories/Library/card-grid/CardGrid.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import CardGrid from "./CardGrid";
+import Card from "../card/Card";
+import ImageCredited from "../image-credited/ImageCredited";
+
+const imageUrl =
```

Comment:
suggestion: 

You could create a file with some different data, instead of using the same. 

I use some different in eventListData.ts, altho i also use the same image. 

Maybe some difference in the design process could be nice since it would be more realistic.

Reply:
I get what you're saying, but, I think this is something we should look into in a more systematic and broader way.